.github/actions/auto-update-seed/action.yaml

```diff
@@ -65,7 +65,7 @@ runs:
       if: steps.check-changes.outputs.seed-changes-generated == 'true'
       shell: bash
       run: |
-        git diff --staged > seed-${{ inputs.generator-name }}-${{ inputs.index }}.patch
+        git diff --staged --binary --full-index > seed-${{ inputs.generator-name }}-${{ inputs.index }}.patch
```

Comment:
can you explain what these two flags do? Is there ANY behavior change for the non-binary files